tests/infra/consortium.py

```diff
@@ -732,13 +732,30 @@ def set_recovery_threshold(self, remote_node, recovery_threshold):
             self.recovery_threshold = recovery_threshold
         return r
 
+    def add_measurement(self, remote_node, platform, measurement):
+        if platform == "sgx":
+            return self.add_new_code(remote_node, measurement)
+        elif platform == "virtual":
+            return self.add_virtual_measurement(remote_node, measurement)
+        elif platform == "snp":
+            return self.add_snp_measurement(remote_node, measurement)
+        else:
+            raise ValueError(f"Unsupported platform {platform}")
```

Comment:
These wrapper functions could live in `code_update.py`, but are plausibly useful for other tests. At the platform-agnostic test code, you want to "update a measurement", but under the hood that has to call a specific governance function to write to a platform-specific table. Other options are available, this is even more copy-paste code, but I think it's "fine".